File: pi_euler/pi.py
# This program give an approach to pi using the solution of Euler to the famous series 1/x²

epsilon = 1E-16
sum = 0
i = 0
term = 1
while term > epsilon :
    i += 1
    term = 1/(i*i)
    sum += term

print(f'The approach of pi using an epsilon of {epsilon} is {(sum*6)**0.5}')

File: pi_euler/pi.f90
program euler
! This program give an approach to pi using the solution of Euler to the famous series 1/x²
implicit none
real*8 epsilon, sum, term, i

epsilon = 1E-16
sum = 0
i = 0
term = 1
do while (term > epsilon)
    i = i + 1
    term = 1/(i*i)
    sum = sum + term
end do

write(*,*) "The approach of pi using an epsilon of"
write(*,*) epsilon
write(*,*) "is"
write(*,*) (sum*6)**0.5
end program
